src/backend/cdb/cdbtimer.c

```diff
@@ -107,17 +121,26 @@ restoreTimers(struct itimers *timers)
 	 */
 	if (timerisset(&timers->rtimer.it_interval) || timerisset(&timers->rtimer.it_value))
 	{
-		err = setitimer(ITIMER_REAL, &timers->rtimer, NULL);
+#ifdef USE_ASSERT_CHECKING
+		err =
```

Comment:
Let's declare err as more local variable to reduce ifdefs
```suggestion
		int err =
```